src/components/v5/common/ActionSidebar/partials/TokenSelect/hooks.tsx

```diff
@@ -69,7 +72,18 @@ export const useTokenSelect = (inputValue: string) => {
       return <SpinnerLoader appearance={{ size: 'small' }} />;
     }
 
-    return tokenData ? (
+    if (readonly && tokenData?.getTokenFromEverywhere) {
+      const selectedToken = tokenData.getTokenFromEverywhere.items?.[0];
+
+      return (
+        <div className="flex items-center gap-2">
+          {selectedToken && <TokenIcon token={selectedToken} size="xxs" />}
+          {selectedToken?.name}
```

Comment:
If the first line fails _(selected token does not exist), then surely the second one won't render either.

So why no add both line under the same availability check ?